src/pages/Classes.js

```diff
@@ -17,6 +17,9 @@ const classExample = `class Animal {
   printName() {
     console.log(this.name)
   }
+
+  printAnimalName() {
+    console.log(`${this.printName()} is an Animal.`)
```

Comment:
@morenoh149 This is helpful, but the `printAnimalName` method is missing a closing curly brace!

Reply:
Also it would print kind of strangely since it's going to call `console.log` both in `printAnimalName` and `printName`